awx/main/migrations/0088_v360_approval_node_notifications.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 2.2.4 on 2019-09-04 18:23
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0087_v360_update_credential_injector_help_text'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='organization',
+            name='notification_templates_approvals',
+            field=models.ManyToManyField(blank=True, related_name='organization_notification_templates_for_approvals', to='main.NotificationTemplate'),
+        ),
+        migrations.AddField(
+            model_name='unifiedjobtemplate',
```

Comment:
I don't think we want *all* `UnifiedJobTemplate`s to have this new relationship, just WFJTs.

You might want to move this:

https://github.com/ansible/awx/pull/4657/files#diff-06484ea63b4597058d47313e1a0e9d6fR396

...so that it's *explicitly* a new field _only_ on the `WorkflowJobTemplate` model definition.